xml/http_servers.xml

```diff
@@ -20,6 +20,15 @@
     <param pos="1" name="service.version"/>
   </fingerprint>
 
+  <fingerprint pattern="(?i)^cherokee(?:/([\d.]+))?$">
```

Comment:
Do you have any example of this banner in lowercase or without the version string?